packages/testsuite/cypress/e2e/mail/test-configuration-subsystem-mail-server.cy.ts

```diff
@@ -0,0 +1,223 @@
+describe("TESTS: Configuration => Subsystems => Mail", () => {
+  const address = [
+    "subsystem",
+    "mail",
+    "mail-session",
+    "my-mail-session",
+    "server",
+    "smtp",
+  ];
+
+  const outBoundSocketBinding = {
+    name: "custom-outbound-socket-binding",
+    host: "localhost",
+    port: "15099",
+  };
+
+  const configurationFormId = "mail-server-form";
+  const mailServerAddFormId = "mail-server-add-form";
+  const credentialsReferenceFormId = "mail-server-credential-reference-add";
+
+  const mailSessionTableId = "mail-server-table";
+  const mailSessionOutboundSocketType = "SMTP";
+
+  const formColumNameClearText = "clear-text";
+  const formColumValueClearText = "ClearTextValue";
+  const credentialsReference = "credential-reference";
+  const formColumNameName = "name";
+  const formColumValueName = "my-mail-session";
+  const formColumNameFrom = "from";
+  const formColumValueDefaultFrom = "DefaultFromValue";
+  const formColumNameJNDI = "jndi-name";
+  const formColumValueDefaultJNDI = "java:jboss/mail/MyJNDI";
+
+  const formColumNameSSL = "ssl";
+  const formColumNameTLS = "tls";
+
+  const formColumNameOutbound = "outbound-socket-binding-ref";
+  const formColumValueOutbound = "custom-outbound-socket-binding";
+
+  const formColumNameUserName = "username";
+  const formColumValueUserName = "Custom username";
+
+  const formColumNamePassword = "password";
+  const formColumValuePassword = "superSecretPassword";
+
+  let managementEndpoint: string;
+
+  before(() => {
+    cy.startWildflyContainer()
+      .wait(2000)
+      .then((result) => {
+        managementEndpoint = result as string;
+        createMailSession();
+        createOutboundSocketBinding();
+      });
+  });
+
+  function createMailSession(): void {
```

Comment:
Since this method (`createOutboundSocketBinding` as well) is also used in the previous test, could you please add it into `commands.ts`? (we could eventually create our own utilities in the `support` folder)

Reply:
new `resource-utils.ts` created with `createMailSession` and `createOutboundSocketBinding`